releasenotes/notes/2.0/add-estimate_duration-method-a35bf8eef4b2f210.yaml

```diff
@@ -1,7 +1,26 @@
 ---
 features_circuits:
   - |
-    Added a new method, :meth:`.QuantumCircuit.estimate_duration`, to compute
-    the estimated duration of a scheduled circuit output from the :mod:`.transpiler`.
-    This should be used if you need an estimate of the full circuit duration instead
-    of the deprecated :attr:`.QuantumCircuit.duration` attribute.
+    Added a new circuit method, :meth:`.QuantumCircuit.estimate_duration`, to estimate the duration of 
+    a scheduled circuit after transpilation. The circuit duration is estimated by finding the 
+    longest path on a scheduled circuit based on the durations provided by a given :class:.Target. 
+    This method only works for simple circuits that do not contain control flow or other classical 
+    feed-forward operations.
+
+    Use this method instead of the deprecated :attr:.QuantumCircuit.duration attribute if you need an estimate of the full circuit duration.
```

Comment:
```suggestion
    Use this method instead of the deprecated :attr:`.QuantumCircuit.duration` attribute if you need an estimate of the full circuit duration.
```